docs/00-contributing/git-workflow.md

```diff
@@ -47,9 +47,9 @@ Resources:
 * [Getting Started with GitHub Desktop](https://help.github.com/desktop/guides/getting-started/)
 * [Contributing to Projects with GitHub Desktop](https://help.github.com/desktop/guides/contributing/)
 
-## Assigning pull requests
+## Assignment of pull requests
```

Comment:
i prefer the less passive voice. but even better: "How to assign a pull request". i'll put in a separate PR for that.

Reply:
Never mind, passive voice is totally appropriate since it's automatic assignment.